### PR TITLE
Concatenate error messages for the same path

### PR DESCRIPTION
**Pull request by msmych**
When there are multiple validation errors for the same path, the response error messages get overrided

This PR fixes that by concatenating error messages for the same path

### CODE REVIEW

File: aleph/tests/test_view_util.py [hunk 1]
```diff
@@ -32,3 +35,33 @@ def test_sanitize_html(self):
...
+    def test_validate_returns_errors_for_paths(self):
+        # given
+        schema = "RoleCreate" # name min length 4, password min length 6
+        data = json.loads('{"name":"Bob","password":"1234","code":"token"}')
+
+        # then
+        with self.assertRaises(BadRequest):
+            validate(data, schema)
+
+        # and
```
Comment (msmych) [inline aleph/tests/test_view_util.py:48]: Don't know how to check that error was raised and verify error details in one run 🤔

**Comment by Rosencrantz** [comment]
Thanks for raising this pull request @msmych we are taking a look now and we'll get back to you with any feedback. Needless to say we really appreciate you taking the time to work on this.

**Review by sunu — COMMENTED** [review]
Hi @msmych, apart from the issue you've mentioned about the tests, this looks great! We can merge this in once you remove the duplicate calls in the tests.

File: aleph/tests/test_view_util.py (hunk 1 above)
Comment (msmych) [reply aleph/tests/test_view_util.py]: @sunu thanks, that worked!

**Comment by msmych** [comment]
@sunu updated

**Review by sunu — APPROVED** [review]

**Comment by sunu** [comment]
Thank you @msmych!

**Pull request merged**